### PR TITLE
Added a recruiter settings page

### PR DESCRIPTION
**Pull request by benmenachem**
Added a recruiter settings page. 
The page allows a recruiter to modify his information. 
As of now the **page can be only accessed directly.**
* After PR #60 will be merged I will create a PR that links the access to the page through the job board.
* After the modification the recruiter will be sent to the success page from PR #60  instead to the job board.

### CODE REVIEW

**Review by YarinBou — APPROVED** [review]
great work ben

**Review by guyy13 — APPROVED** [review]
you're too good at this

**Review by yaelerdr — APPROVED** [review]

**Review by eldaraharonson — APPROVED** [review]

File: recruiter/views.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+from recruiter.models import Recruiter
+
+
+class UpdateRecruiterSettings(UserPassesTestMixin, UpdateView):
+    model = Recruiter
+    fields = ('name', 'company', 'email', 'phone_number')
+    template_name = 'recruiter_account_settings.html'
+    success_url = '/'
+
+    def test_func(self):
```
Comment (guy9050) [inline recruiter/views.py:15]: where is this test used?

Comment (benmenachem) [reply recruiter/views.py:15]: when a recruiter enters his settings page the test verifies that he is logging into his account settings page and that he doesn't have an access to any other recruiter's settings page

**Review by YarinBou — APPROVED** [review]

**Pull request merged**